<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSysParaWgt</class>
 <widget class="QWidget" name="CSysParaWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSysParaWgt</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>395</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="m_HorLayout">
    <item>
     <widget class="QPushButton" name="m_BtnAdd">
      <property name="text">
       <string>增加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_BtnDel">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_BtnOk">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_BtnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="m_pSysParaTab">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>671</width>
     <height>451</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>描述</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>数值</string>
    </property>
   </column>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
